.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  "extends": [
+    "react-app",
+    "prettier/@typescript-eslint",
+    "plugin:prettier/recommended"
+  ],
+  "settings": {
+    "react": {
+      "version": "detect"
+    }
+  },
+  "rules": {
+    "array-callback-return": "off"
```

Comment:
Why?

Reply:
Because we use this syntax and it was complaining (and don't see any good reason for this rule 😁)

ref: https://eslint.org/docs/rules/array-callback-return

Reply:
If you forget to `return` inside `Array.prototype.map`, or `Array.prototype.reduce`, you end up with an array of `undefined`. Could be dangerous, no? 🤔

Reply:
I can maybe give a try for `["error", { allowImplicit: true }]`

Reply:
> If you forget to `return` inside `Array.prototype.map`, or `Array.prototype.reduce`, you end up with an array of `undefined`. Could be dangerous, no? 🤔

If you finish with an array of `undefined` and you commit this without noticing it, YOU are dangerous (everything should be tested ^^)

Reply:
But looking a bit more where it was failing, I can just transform my `.map` and `.forEach` and everything follow, so let's go back on the "recommended" setup ^^